packages/arb-token-bridge-ui/scripts/generateOrbitChainsToMonitor.ts

```diff
@@ -1,45 +1,14 @@
 import fs from 'fs'
-import { L2Network } from '@arbitrum/sdk'
 import { getOrbitChains } from '../src/util/orbitChainsList'
-import { getExplorerUrl, rpcURLs } from '../src/util/networks'
-
-interface ChildNetwork extends L2Network {
```

Comment:
Refactored all the common interfaces and utility functions out of this script into a dedicated `utils` file.